src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ChatEntry from './ChatEntry'
+
+function ChatLog(props) {
+    const data = props.entries;  
+    return (
+    <>
+    {data.map((entries)=>{
+
+    return (
+    
+    <ChatEntry
+    toggleLike={props.toggleLike}
+    key={entries.id}
+    id={entries.id}
+    sender={entries.sender} 
+    body={entries.body} 
+    timeStamp={entries.timeStamp}
+    isLiked={entries.liked}/>
+    )})
+};
+    </>
+)};
+
```

Comment:
Remember to put your `props-types` checking for this one. You could use the `arrayOf` and then `shape` functionality to be very explicit about the data structure you are expecting to pass through here.